rust/tvm-rt/src/to_function.rs

```diff
@@ -232,13 +270,18 @@ macro_rules! impl_typed_and_to_function {
             }
 
             #[allow(non_snake_case)]
-            fn call(handle: *mut Self::Handle, args: Vec<ArgValue<'static>>) -> Result<RetValue>
+            fn call<'a>(handle: *mut Self::Handle, args: Vec<ArgValue<'a>>) -> Result<RetValue>
```

Comment:
I think either this must be unsafe or into_raw() must be unsafe since you could do something like:
```
let handle = f.into_raw();
drop(f);
F::call(f.into_raw(), args);
```

Reply:
Poked around at this with tests and the Box that is added makes it safe